### ISSUE TITLE
Detection rates?

### ISSUE THREAD

**Issue by tokyoneon** [mado-95/madwin#1]
I saw your article on Null Byte, but don't have the time at the moment to properly test `madwin` myself. I'm curious to know what kind of detection rates the generated payloads get? Have you tested them against Windows Defender and/or antivirus software?

**Comment by mado-95** [comment]
Sorry for the late reply! I threw away the article briefly after a bit and decided to rewrite some time later, so I didn't think anyone saw. Anyway, the file is executing fine against a fully patched Windows Defender on Win 10 Pro. Since nodistribute is broken and virustotal distributes, I can't exactly check other AVs without lining up madwin for the chopping block. Something I noticed is that in one particular run (back when the source generator only had like 8~ padder calls), Windows suggested submitting the file, but still let it execute and communicate over the network fine. Adding a few more padder calls seemed to have fixed this. Still though, it's alarming even Defender is moving towards heuristic analysis. 

TL;DR you should be fine for Defender. Probably the same for horrendously outdated suites like Sophos Endpoint. If, however, you're hacking into a decently protected system I'd recommend Phantom-Evasion, as I love the decoy process behaviour, or just write a backdoor of your own.

PS; love your Armor software! I always wanted a way to obfuscate my bash -i invocations when backdooring OSX.
